src/init.cpp

```diff
@@ -1777,7 +1780,11 @@ bool AppInitMain(const util::Ref& context, NodeContext& node, interfaces::BlockA
     node.chainman = &g_chainman;
     ChainstateManager& chainman = *Assert(node.chainman);
 
-    node.peer_logic.reset(new PeerLogicValidation(node.connman.get(), node.banman.get(), *node.scheduler, chainman, *node.mempool, args.GetBoolArg("-enablebip61", DEFAULT_ENABLE_BIP61)));
+    node.peer_logic.reset(new PeerLogicValidation(
+        node.connman.get(), node.banman.get(), *node.scheduler, chainman, *node.mempool, *llmq::quorumBlockProcessor,
+        *llmq::quorumDKGSessionManager, *llmq::quorumManager, *llmq::quorumSigSharesManager, *llmq::quorumSigningManager,
+        *llmq::chainLocksHandler, *llmq::quorumInstantSendManager, args.GetBoolArg("-enablebip61", DEFAULT_ENABLE_BIP61))
+    );
```

Comment:
The former solution is better imo since I want to dereference as early as possible to encourage fast-failing in case of developer error in the codebase versus a potential failure being nested deep inside an edge case.

Will be modifying each commit to pass the `unique_ptr` reference. I don't like having to pass everything and create a clunky argument list (though there are cases where this might be justified, `PeerLogicValidation` doesn't seem to be such a case) so the creation of an LLMQ context should take care of that but that's within the purview of a future PR. 

The intent would be to have pointer initialization checks there as well.